src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -389,11 +381,9 @@ public CaseWorkerProfile createCaseWorkerProfile(CaseWorkersProfileCreationReque
                     updateUserRolesInIdam(cwrdProfileRequest, userProfileCreationResponse.getIdamId());
                 }
             } else {
-                log.error("{}::UP response missing body", loggingComponentName);
+                validationServiceFacade.logFailures(UP_MISSING_RESPONSE_BODY, cwrdProfileRequest.getRowId());
+                log.error("{}::" + UP_MISSING_RESPONSE_BODY, loggingComponentName);
```

Comment:
Can we use like below
`log.error("{}::{}" , loggingComponentName, UP_MISSING_RESPONSE_BODY);`